iphone/metadata/tr/release_notes.txt

```diff
@@ -1,11 +1,13 @@
-Kalbimiz depremden etkilenen herkesle birlikte
-
-Yaşamını yitirenlere Allah'tan rahmet, yaralılara acil şifalar dileriz. Tüm milletimizin başı sağ olsun.
-
-#GeçmişOlsunTürkiye
-
-Deprem bölgesindekilere yardımcı olabilmek adına Güneydoğu Anadolu'daki harita verileri güncellendi.
-
-Organic Maps ile şarjınızı korurken internetsiz olarak haritayı kullanabilirsiniz. Gerekli durumlarda Organic Maps ayarlarında bulunan "maksimum güç tasarrufunu" açmayı unutmayın.
-
-Daha fazla bilgi: omaps.app/tr/news
+• New OpenStreetMap data as of February 25
+• Fixed freeze on iPad when a keyboard is connected
+• Fixed unnecessary double click on the route button
+• Removed duplicated floors and zip code cells
+• Enabled OpenStreetMap editor notes for all selected places
+• Fixed and improved search
+• Split vegan and vegetarian cuisines in search
+• Fixed styles for airports, landuses, waterways, and hatching fill areas
+• Added healthcare specialists: alternative, audiologist, blood_donation, optometrist, podiatrist, psychotherapist, sample_collection, speech_therapist
+• Added communication towers and solar, wind, gas, and hydropower generators
+• Improved Arabic, Norwegian, Portuguese, and Turkish translations
+
+Follow our news at omaps.app/news
```

Comment:
```suggestion
• 25 Şubat tarihine ait yeni OpenStreetMap harita verileri
• iPad'lerde bir klavye bağlandığında oluşan donma sorunu düzeltildi
• Rota oluşturma butonundaki gereksiz ikinci tıklama kaldırıldı
• Yinelenen kat ve posta kodu hücreleri kaldırıldı
• Seçilen tüm yerler için OpenStreetMap editör notları etkinleştirildi
• Arama düzeltildi ve iyileştirildi
• Arama sayfasında vegan ve vejetaryen yemek kategorileri ayrıldı
• Havaalanları, kara yolları, su yolları ve tarama dolgu alanları stilleri düzeltildi
• Sağlık alanında bazı uzmanlıklar eklendi: alternatif tıb, odyolog, kan bağışı, optometrist, ayak hastalıkları uzmanı, psikoterapist, numune toplama, konuşma terapisti
• Haberleşme kulesi ve güneş, rüzgar, gaz, hidroelektrik jeneratörleri eklendi
• Çeviriler iyileştirildi

Gelişmelerimizi omaps.app/tr/news üzerinden takip edebilirsiniz
```

Reply:
Can you edit the translation for ```Fixed styles for airports, landuses, waterways, and hatching fill areas```
to
```• Havaalanları, landuse (arazi türü etiketi), waterways (su kategori etiketi) ve tarama dolgu alanları stilleri düzeltildi```

current translation: ```• Havaalanları, kara yolları, su yolları ve tarama dolgu alanları stilleri düzeltildi```